src/temp.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+</head>
+<body>
+<header>
+    <div class="link before">
+        <a href="#" class="nav-link indent active"><h1>PORTFOLIO</h1></a>
+        <a href="#" class="nav-link">ABOUT</a>
+    </div>
+    <div class="pointy-thingy"></div>
+    <div class="logo"><img src="Logo.svg" alt="Logo"></div>
+    <div class="link after">
+        <a href="#" class="indent nav-link">BLOG</a>
+        <a href="#" class="nav-link">GET IN TOUCH</a>
+    </div>
+</header>
+<div class="portfolio">
```

Comment:
Use semantic HTML tag here. If you have header and footer tags, it would be consistent to use a semantic tag for the content itself.